conf/backup_method

```diff
@@ -5,13 +5,18 @@ app="${0#"./05-"}"
 app="${app%"_app"}"
 
 BORG_PASSPHRASE="$(yunohost app setting $app passphrase)"
-repo="$(yunohost app setting $app repository)"   #$4
+repos="$(yunohost app setting $app repository)"   #$4
 if ssh-keygen -F "__SERVER__" >/dev/null ; then
```

Comment:
if there are several repos, i guess we should adapt this part to. __SERVER__ is extracted from repository var if i remember well. And i think we should improve this part by deducing server from $repository (instead of hradcoding it during install script...)